CHANGELOG.md

```diff
@@ -4,16 +4,48 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## 1.5.0 (unreleased)
+## 1.5.0 (2019-10-31)
+
+This release adds major new features and bugfixes since the 1.4.2 release.
+We deem it moderate priority for upgrading.
+
+This release adds compression of hypertable chunks as a major new feature.
+
+This release also adds support for parallel query coordination to the
```

Comment:
Maybe something like:

"This release also adds support for basic data tiering by supporting the migration of chunks between tablespaces, as well as support for parallel..."